### PR TITLE
build: patch around qt duplicate symbol issue

### PR DESCRIPTION
**Pull request by fanquake**
This is currently causing the same failure in two different PRs:

#24391: build: stop overriding user autoconf flags
```bash
duplicate symbol 'lcQpaFonts()' in:
    /tmp/cirrus-ci-build/depends/x86_64-apple-darwin/lib/libQt5ThemeSupport.a(qgenericunixthemes.o)
    /tmp/cirrus-ci-build/depends/x86_64-apple-darwin/lib/libQt5FontDatabaseSupport.a(qfontengine_coretext.o)
ld: 1 duplicate symbol for architecture x86_64
```

#21778: build: LLVM 14 & LLD based macOS toolchain
```bash
x86_64-apple-darwin-ld: error: duplicate symbol: __Z10lcQpaFontsv
>>> defined in /tmp/cirrus-ci-build/depends/x86_64-apple-darwin/lib/libQt5ThemeSupport.a(qgenericunixthemes.o)
>>> defined in /tmp/cirrus-ci-build/depends/x86_64-apple-darwin/lib/libQt5FontDatabaseSupport.a(qfontengine_coretext.o)
```

Guix Build (x86_64):
```bash
82dd305045fa908d3a5909c8f7827e4ab607281b84706e0722d32e4c1c507cef  guix-build-a24ef27cfe6e/output/aarch64-linux-gnu/SHA256SUMS.part
dadef04a291a1e371cac7eef0056948180fcad86ba77f9fc7b844b389461bbe1  guix-build-a24ef27cfe6e/output/aarch64-linux-gnu/bitcoin-a24ef27cfe6e-aarch64-linux-gnu-debug.tar.gz
67842aef90a56be203388a6838b95d6f472d018fdbf399b12b9e0b577e101651  guix-build-a24ef27cfe6e/output/aarch64-linux-gnu/bitcoin-a24ef27cfe6e-aarch64-linux-gnu.tar.gz
bef26de8cdd47853c1bca34e2cf002a9734394a7a3e027aba1b713738b4a3794  guix-build-a24ef27cfe6e/output/arm-linux-gnueabihf/SHA256SUMS.part
5198227d1b7801032f358da348bc577a4d99273b0ecca50b83ad1b3d5d2f9d92  guix-build-a24ef27cfe6e/output/arm-linux-gnueabihf/bitcoin-a24ef27cfe6e-arm-linux-gnueabihf-debug.tar.gz
97656b94a4b8bedd87a8a2d11e032d01f6cb22fa22f26e047deab9fc70886733  guix-build-a24ef27cfe6e/output/arm-linux-gnueabihf/bitcoin-a24ef27cfe6e-arm-linux-gnueabihf.tar.gz
ea0b7fa4458d36c349d7b56813c75a1957bb904945265bd26f970a5b6f8a1817  guix-build-a24ef27cfe6e/output/arm64-apple-darwin/SHA256SUMS.part
f7741dc1baf0e59713f42fef8b53e613fef62abbeef9ca5c8187b04b7a81054f  guix-build-a24ef27cfe6e/output/arm64-apple-darwin/bitcoin-a24ef27cfe6e-arm64-apple-darwin-unsigned.dmg
7d27ebc99af2a35ee21736dc13adf76c5e646a3b5c00cf6a4bf56077b1ce0db4  guix-build-a24ef27cfe6e/output/arm64-apple-darwin/bitcoin-a24ef27cfe6e-arm64-apple-darwin-unsigned.tar.gz
ae47461a03cff74d8e9e575c52148d1378ed883f41557cdf57f626532adfd60e  guix-build-a24ef27cfe6e/output/arm64-apple-darwin/bitcoin-a24ef27cfe6e-arm64-apple-darwin.tar.gz
14f55113414fa633e4499cd83ebed9f1f4d9a36e0ab2ec550cbba159d00d1de8  guix-build-a24ef27cfe6e/output/dist-archive/bitcoin-a24ef27cfe6e.tar.gz
a2834efc9b7ac1545e442f5b527b1a739713b399c3591cfa4ac07d30a858017f  guix-build-a24ef27cfe6e/output/powerpc64-linux-gnu/SHA256SUMS.part
720120e641fe527dbe1f6979e3ed0222238f79a0cfac067838bc91739aa3f2d9  guix-build-a24ef27cfe6e/output/powerpc64-linux-gnu/bitcoin-a24ef27cfe6e-powerpc64-linux-gnu-debug.tar.gz
e150946434a4f0b59b7dfdec65d02a2db4290b3e7f5eade1141905ef56b7587b  guix-build-a24ef27cfe6e/output/powerpc64-linux-gnu/bitcoin-a24ef27cfe6e-powerpc64-linux-gnu.tar.gz
1255c0e22087042f2262acc3f7d043659a9a2cbd0767b65ba355f89bc4589915  guix-build-a24ef27cfe6e/output/powerpc64le-linux-gnu/SHA256SUMS.part
43152dcfcff13d60c5f6fbfaed328643e6a18d43f1f0935742732080e142e40d  guix-build-a24ef27cfe6e/output/powerpc64le-linux-gnu/bitcoin-a24ef27cfe6e-powerpc64le-linux-gnu-debug.tar.gz
41ec8e457921ecf4241209015441bcf6ced53e1430f5637aef0ed837de175dfb  guix-build-a24ef27cfe6e/output/powerpc64le-linux-gnu/bitcoin-a24ef27cfe6e-powerpc64le-linux-gnu.tar.gz
5edeefb9479d14ff8b2f6c40390ae6a8995d42238be56c1edd2c462d065fb1e7  guix-build-a24ef27cfe6e/output/riscv64-linux-gnu/SHA256SUMS.part
f037d4cc378da0c00b10f2960cb853377b2c0710aba4ed17c8f10ff9e8ab2cd8  guix-build-a24ef27cfe6e/output/riscv64-linux-gnu/bitcoin-a24ef27cfe6e-riscv64-linux-gnu-debug.tar.gz
793d0f8710e300adb6aff2e289370fb71c60d5d844970161898c359f3dea0f2f  guix-build-a24ef27cfe6e/output/riscv64-linux-gnu/bitcoin-a24ef27cfe6e-riscv64-linux-gnu.tar.gz
caeed20b8b301abdd32cb82c290ef77b283eb7b9f341254cdcf846f6d85365a9  guix-build-a24ef27cfe6e/output/x86_64-apple-darwin/SHA256SUMS.part
5f02de0e7149b70b6fe612cc382e27c3e8691a2c8d05730604ea92410843d62c  guix-build-a24ef27cfe6e/output/x86_64-apple-darwin/bitcoin-a24ef27cfe6e-x86_64-apple-darwin-unsigned.dmg
0f5a5041a12061e4e9f5fcfc803cd2a98636d964845a76713d83575caef5a4f6  guix-build-a24ef27cfe6e/output/x86_64-apple-darwin/bitcoin-a24ef27cfe6e-x86_64-apple-darwin-unsigned.tar.gz
adc74b896eba9eb6b6ef5da0d909f2d3c529a23b55fff4c8af4465b277b46217  guix-build-a24ef27cfe6e/output/x86_64-apple-darwin/bitcoin-a24ef27cfe6e-x86_64-apple-darwin.tar.gz
57a56059e4054e5becdbc36a6765f3b209535a78d4fa3f8a932d65e6b12d9c05  guix-build-a24ef27cfe6e/output/x86_64-linux-gnu/SHA256SUMS.part
5c163f6c9936307b139fcc91e9a129e144c7961b44e11b245b0c9402b6b919a8  guix-build-a24ef27cfe6e/output/x86_64-linux-gnu/bitcoin-a24ef27cfe6e-x86_64-linux-gnu-debug.tar.gz
7c1f9370fe8e26e3363198f3cd3f1d817662bc944f873138bcff997cda2d3c25  guix-build-a24ef27cfe6e/output/x86_64-linux-gnu/bitcoin-a24ef27cfe6e-x86_64-linux-gnu.tar.gz
0708072d2c81ed5e7d54bfec350d8737931e13bd9291bdb72925c096be33f186  guix-build-a24ef27cfe6e/output/x86_64-w64-mingw32/SHA256SUMS.part
0b9b931fb4902a2779a193ca96fee01d1790644a004cd8898e24f008799d1a24  guix-build-a24ef27cfe6e/output/x86_64-w64-mingw32/bitcoin-a24ef27cfe6e-win64-debug.zip
d19c8cbb2b3359ebf0b66dada69f3a88eb41c0945c0e2b847bc2684f1c3d1985  guix-build-a24ef27cfe6e/output/x86_64-w64-mingw32/bitcoin-a24ef27cfe6e-win64-setup-unsigned.exe
5797bb51c61b2857c0912e205bc886a29bc537076da00bba4c3e27c479d23c9c  guix-build-a24ef27cfe6e/output/x86_64-w64-mingw32/bitcoin-a24ef27cfe6e-win64-unsigned.tar.gz
131d9aff0d9c72f1d27cac728b7fdd42f6e961214eec330fc38e1d644869b6f8  guix-build-a24ef27cfe6e/output/x86_64-w64-mingw32/bitcoin-a24ef27cfe6e-win64.zip
```

Guix Build (arm64):
```bash
fef112513bde9d33b071efd0e1c3965b10103c7f8c2c66838b6e2f0556bb62b2  guix-build-a24ef27cfe6e/output/arm-linux-gnueabihf/SHA256SUMS.part
71b8117e6558fc455235a172b65f882b1d2ee8bcebc456f97ceafaa109c683ee  guix-build-a24ef27cfe6e/output/arm-linux-gnueabihf/bitcoin-a24ef27cfe6e-arm-linux-gnueabihf-debug.tar.gz
25df44270fc5495967509dd36455ad70179f30b9d2819f6a86a72bbabb7a0396  guix-build-a24ef27cfe6e/output/arm-linux-gnueabihf/bitcoin-a24ef27cfe6e-arm-linux-gnueabihf.tar.gz
918ce9c69609f4cf07fdb2de699c54db4ad39dc0e79eeb9ea1b7ef7333bc7d5c  guix-build-a24ef27cfe6e/output/arm64-apple-darwin/SHA256SUMS.part
63a6082ed74497c957309d72e517905ac1071c45527241c8d5d08327af00e3b4  guix-build-a24ef27cfe6e/output/arm64-apple-darwin/bitcoin-a24ef27cfe6e-arm64-apple-darwin-unsigned.dmg
dce37e97b56c9612f7fed92c58a0f9ba58a7829db2dabcd6fea56935fa4aa7dd  guix-build-a24ef27cfe6e/output/arm64-apple-darwin/bitcoin-a24ef27cfe6e-arm64-apple-darwin-unsigned.tar.gz
056aaa181c8d6f6ae6cf9e8c1d3d9344c31ed4b73fd41d7fe2b89761d2a8ca7e  guix-build-a24ef27cfe6e/output/arm64-apple-darwin/bitcoin-a24ef27cfe6e-arm64-apple-darwin.tar.gz
14f55113414fa633e4499cd83ebed9f1f4d9a36e0ab2ec550cbba159d00d1de8  guix-build-a24ef27cfe6e/output/dist-archive/bitcoin-a24ef27cfe6e.tar.gz
e16e0f4fedf8761f220098c1986c0d53f4f051362ba4ff44d89a350a62f3fd3a  guix-build-a24ef27cfe6e/output/powerpc64-linux-gnu/SHA256SUMS.part
ae804d7a23bda9dc196200cd9d4caf0bd5d0611597a3e40da1dc2b27dee0effd  guix-build-a24ef27cfe6e/output/powerpc64-linux-gnu/bitcoin-a24ef27cfe6e-powerpc64-linux-gnu-debug.tar.gz
82b274e6bbca6f5c2b70809b9aee7831dcefe6553011aabef804ca0a531c54d8  guix-build-a24ef27cfe6e/output/powerpc64-linux-gnu/bitcoin-a24ef27cfe6e-powerpc64-linux-gnu.tar.gz
15031a61fbb3cc9495f274a85241aa4708aef3564a66faaaccfd35fcb51e5f3c  guix-build-a24ef27cfe6e/output/powerpc64le-linux-gnu/SHA256SUMS.part
302041c21942edb8c0de8aa4af6105338c82ffead5ad57947c0fdd99f1f66ce7  guix-build-a24ef27cfe6e/output/powerpc64le-linux-gnu/bitcoin-a24ef27cfe6e-powerpc64le-linux-gnu-debug.tar.gz
0f6b18d72109efd249abdae611d57b94a725863a4e8ab91b07b9199d9e5e3e86  guix-build-a24ef27cfe6e/output/powerpc64le-linux-gnu/bitcoin-a24ef27cfe6e-powerpc64le-linux-gnu.tar.gz
d3e84416a8dd90427d028390357cb9fbf98aabc1e97210afdcb26ff7b7656b93  guix-build-a24ef27cfe6e/output/riscv64-linux-gnu/SHA256SUMS.part
564e353892b5952889e62f946fe6045f4b348fa66712f7cb06262aca8bd7fbc8  guix-build-a24ef27cfe6e/output/riscv64-linux-gnu/bitcoin-a24ef27cfe6e-riscv64-linux-gnu-debug.tar.gz
29fc019a6c96be6a6f9190e2661be9b4c4d2da7b530ee8815be145cdfa11902a  guix-build-a24ef27cfe6e/output/riscv64-linux-gnu/bitcoin-a24ef27cfe6e-riscv64-linux-gnu.tar.gz
3a9e1d03eb6691138f193da68e355638d4db4646b3f674ea1645358cf55a49f2  guix-build-a24ef27cfe6e/output/x86_64-apple-darwin/SHA256SUMS.part
5f02de0e7149b70b6fe612cc382e27c3e8691a2c8d05730604ea92410843d62c  guix-build-a24ef27cfe6e/output/x86_64-apple-darwin/bitcoin-a24ef27cfe6e-x86_64-apple-darwin-unsigned.dmg
8923d5f2b08b64fd53796db4ef90a631b8b1c15908971ae334d7ba35437e0f19  guix-build-a24ef27cfe6e/output/x86_64-apple-darwin/bitcoin-a24ef27cfe6e-x86_64-apple-darwin-unsigned.tar.gz
adc74b896eba9eb6b6ef5da0d909f2d3c529a23b55fff4c8af4465b277b46217  guix-build-a24ef27cfe6e/output/x86_64-apple-darwin/bitcoin-a24ef27cfe6e-x86_64-apple-darwin.tar.gz
faff9251c2d7cdd4336ab23baa34b9c52083018d5302eb3e7aa807926ef6b2b8  guix-build-a24ef27cfe6e/output/x86_64-linux-gnu/SHA256SUMS.part
92c72070e2c95aad4ad16434b7eadef7ebdbbaa1de262ad968d4fa8396e801c4  guix-build-a24ef27cfe6e/output/x86_64-linux-gnu/bitcoin-a24ef27cfe6e-x86_64-linux-gnu-debug.tar.gz
259f6213e0ecb0f4d01d2c106142602e7772c1b09a15ff6cf395d2f9be571787  guix-build-a24ef27cfe6e/output/x86_64-linux-gnu/bitcoin-a24ef27cfe6e-x86_64-linux-gnu.tar.gz
22cf628aad35d18206a87514a9833acc0af79572ecba847aa59c2b2dec1be799  guix-build-a24ef27cfe6e/output/x86_64-w64-mingw32/SHA256SUMS.part
29cbe82b770d6db0ab295ff0ac2614584de5fc5cc79dc4f8f8d9918f359795dc  guix-build-a24ef27cfe6e/output/x86_64-w64-mingw32/bitcoin-a24ef27cfe6e-win64-debug.zip
d19c8cbb2b3359ebf0b66dada69f3a88eb41c0945c0e2b847bc2684f1c3d1985  guix-build-a24ef27cfe6e/output/x86_64-w64-mingw32/bitcoin-a24ef27cfe6e-win64-setup-unsigned.exe
5797bb51c61b2857c0912e205bc886a29bc537076da00bba4c3e27c479d23c9c  guix-build-a24ef27cfe6e/output/x86_64-w64-mingw32/bitcoin-a24ef27cfe6e-win64-unsigned.tar.gz
56ac734a7007d5565bee6147bc4219e0750210dae8188b602f12d4243a437fac  guix-build-a24ef27cfe6e/output/x86_64-w64-mingw32/bitcoin-a24ef27cfe6e-win64.zip
```

### CODE REVIEW

**Comment by hebasto** [comment]
Concept ACK.

**Review by hebasto — CHANGES_REQUESTED** [review]
Approach NACK f530cc01950f3de06de9dea57d2a75782957db22

Why do not backport the upstream [commit](https://github.com/qt/qtbase/commit/ab01885e48873fb2ad71841a3f1627fe4d9cd835):

```
$ cat depends/patches/qt/duplicate_lcqpafonts.patch 
QtGui: Fix duplication of logging category lcQpaFonts

Move it to qplatformfontdatabase.h.

Upstream commit:
 - Qt 6.0: ab01885e48873fb2ad71841a3f1627fe4d9cd835

--- a/qtbase/src/gui/text/qplatformfontdatabase.cpp
+++ b/qtbase/src/gui/text/qplatformfontdatabase.cpp
@@ -52,6 +52,8 @@
 
 QT_BEGIN_NAMESPACE
 
+Q_LOGGING_CATEGORY(lcQpaFonts, "qt.qpa.fonts")
+
 void qt_registerFont(const QString &familyname, const QString &stylename,
                      const QString &foundryname, int weight,
                      QFont::Style style, int stretch, bool antialiased,

--- a/qtbase/src/gui/text/qplatformfontdatabase.h
+++ b/qtbase/src/gui/text/qplatformfontdatabase.h
@@ -50,6 +50,7 @@
 //
 
 #include <QtGui/qtguiglobal.h>
+#include <QtCore/qloggingcategory.h>
 #include <QtCore/QString>
 #include <QtCore/QStringList>
 #include <QtCore/QList>
@@ -62,6 +63,7 @@
 
 QT_BEGIN_NAMESPACE
 
+Q_DECLARE_LOGGING_CATEGORY(lcQpaFonts)
 
 class QWritingSystemsPrivate;
 

--- a/qtbase/src/platformsupport/fontdatabases/mac/qfontengine_coretext.mm
+++ b/qtbase/src/platformsupport/fontdatabases/mac/qfontengine_coretext.mm
@@ -86,8 +86,6 @@
 
 QT_BEGIN_NAMESPACE
 
-Q_LOGGING_CATEGORY(lcQpaFonts, "qt.qpa.fonts")
-
 static float SYNTHETIC_ITALIC_SKEW = std::tan(14.f * std::acos(0.f) / 90.f);
 
 bool QCoreTextFontEngine::ct_getSfntTable(void *user_data, uint tag, uchar *buffer, uint *length)

--- a/qtbase/src/platformsupport/fontdatabases/mac/qfontengine_coretext_p.h
+++ b/qtbase/src/platformsupport/fontdatabases/mac/qfontengine_coretext_p.h
@@ -64,8 +64,6 @@
 
 QT_BEGIN_NAMESPACE
 
-Q_DECLARE_LOGGING_CATEGORY(lcQpaFonts)
-
 class QCoreTextFontEngine : public QFontEngine
 {
     Q_GADGET

--- a/qtbase/src/platformsupport/fontdatabases/windows/qwindowsfontdatabase.cpp
+++ b/qtbase/src/platformsupport/fontdatabases/windows/qwindowsfontdatabase.cpp
@@ -68,8 +68,6 @@
 
 QT_BEGIN_NAMESPACE
 
-Q_LOGGING_CATEGORY(lcQpaFonts, "qt.qpa.fonts")
-
 #ifndef QT_NO_DIRECTWRITE
 // ### fixme: Consider direct linking of dwrite.dll once Windows Vista pre SP2 is dropped (QTBUG-49711)
 

--- a/qtbase/src/platformsupport/fontdatabases/windows/qwindowsfontdatabase_p.h
+++ b/qtbase/src/platformsupport/fontdatabases/windows/qwindowsfontdatabase_p.h
@@ -63,8 +63,6 @@
 
 QT_BEGIN_NAMESPACE
 
-Q_DECLARE_LOGGING_CATEGORY(lcQpaFonts)
-
 class QWindowsFontEngineData
 {
     Q_DISABLE_COPY_MOVE(QWindowsFontEngineData)
```
?

This approach does not introduce new symbols, and it is more manageable during Qt update in the future.

FWIW, tested it with #21778.

**Review by hebasto — APPROVED** [review]
~ACK e351412ff0fc24b797ec2623812f0454f260b586~

EDIT: unfortunately, CentOS CI fails (

nit: missed EOL

Screenshot from 2022-03-31 16-38-42

**Comment by hebasto** [comment]
> FWIW, tested it with #21778.

Sorry, was building only for macOS, did not test linux builds.

**Comment by fanquake** [comment]
> Sorry, was building only for macOS, did not test linux builds.

Ok. Reverted back to my original change.

**Comment by hebasto** [comment]
> > FWIW, tested it with #21778.
> 
> Sorry, was building only for macOS, did not test linux builds.

A fixed [patch](https://github.com/hebasto/bitcoin/commit/2e3868e5f77e235f9e19c25df63dd9f52a4ea5c5) has been suggested in https://github.com/hebasto/bitcoin/commits/pr21778/0331-test.

**Comment by hebasto** [comment]
#### Guix builds `x86_64`:
```
$ find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
82dd305045fa908d3a5909c8f7827e4ab607281b84706e0722d32e4c1c507cef  guix-build-a24ef27cfe6e/output/aarch64-linux-gnu/SHA256SUMS.part
dadef04a291a1e371cac7eef0056948180fcad86ba77f9fc7b844b389461bbe1  guix-build-a24ef27cfe6e/output/aarch64-linux-gnu/bitcoin-a24ef27cfe6e-aarch64-linux-gnu-debug.tar.gz
67842aef90a56be203388a6838b95d6f472d018fdbf399b12b9e0b577e101651  guix-build-a24ef27cfe6e/output/aarch64-linux-gnu/bitcoin-a24ef27cfe6e-aarch64-linux-gnu.tar.gz
bef26de8cdd47853c1bca34e2cf002a9734394a7a3e027aba1b713738b4a3794  guix-build-a24ef27cfe6e/output/arm-linux-gnueabihf/SHA256SUMS.part
5198227d1b7801032f358da348bc577a4d99273b0ecca50b83ad1b3d5d2f9d92  guix-build-a24ef27cfe6e/output/arm-linux-gnueabihf/bitcoin-a24ef27cfe6e-arm-linux-gnueabihf-debug.tar.gz
97656b94a4b8bedd87a8a2d11e032d01f6cb22fa22f26e047deab9fc70886733  guix-build-a24ef27cfe6e/output/arm-linux-gnueabihf/bitcoin-a24ef27cfe6e-arm-linux-gnueabihf.tar.gz
ea0b7fa4458d36c349d7b56813c75a1957bb904945265bd26f970a5b6f8a1817  guix-build-a24ef27cfe6e/output/arm64-apple-darwin/SHA256SUMS.part
f7741dc1baf0e59713f42fef8b53e613fef62abbeef9ca5c8187b04b7a81054f  guix-build-a24ef27cfe6e/output/arm64-apple-darwin/bitcoin-a24ef27cfe6e-arm64-apple-darwin-unsigned.dmg
7d27ebc99af2a35ee21736dc13adf76c5e646a3b5c00cf6a4bf56077b1ce0db4  guix-build-a24ef27cfe6e/output/arm64-apple-darwin/bitcoin-a24ef27cfe6e-arm64-apple-darwin-unsigned.tar.gz
ae47461a03cff74d8e9e575c52148d1378ed883f41557cdf57f626532adfd60e  guix-build-a24ef27cfe6e/output/arm64-apple-darwin/bitcoin-a24ef27cfe6e-arm64-apple-darwin.tar.gz
14f55113414fa633e4499cd83ebed9f1f4d9a36e0ab2ec550cbba159d00d1de8  guix-build-a24ef27cfe6e/output/dist-archive/bitcoin-a24ef27cfe6e.tar.gz
a2834efc9b7ac1545e442f5b527b1a739713b399c3591cfa4ac07d30a858017f  guix-build-a24ef27cfe6e/output/powerpc64-linux-gnu/SHA256SUMS.part
720120e641fe527dbe1f6979e3ed0222238f79a0cfac067838bc91739aa3f2d9  guix-build-a24ef27cfe6e/output/powerpc64-linux-gnu/bitcoin-a24ef27cfe6e-powerpc64-linux-gnu-debug.tar.gz
e150946434a4f0b59b7dfdec65d02a2db4290b3e7f5eade1141905ef56b7587b  guix-build-a24ef27cfe6e/output/powerpc64-linux-gnu/bitcoin-a24ef27cfe6e-powerpc64-linux-gnu.tar.gz
1255c0e22087042f2262acc3f7d043659a9a2cbd0767b65ba355f89bc4589915  guix-build-a24ef27cfe6e/output/powerpc64le-linux-gnu/SHA256SUMS.part
43152dcfcff13d60c5f6fbfaed328643e6a18d43f1f0935742732080e142e40d  guix-build-a24ef27cfe6e/output/powerpc64le-linux-gnu/bitcoin-a24ef27cfe6e-powerpc64le-linux-gnu-debug.tar.gz
41ec8e457921ecf4241209015441bcf6ced53e1430f5637aef0ed837de175dfb  guix-build-a24ef27cfe6e/output/powerpc64le-linux-gnu/bitcoin-a24ef27cfe6e-powerpc64le-linux-gnu.tar.gz
5edeefb9479d14ff8b2f6c40390ae6a8995d42238be56c1edd2c462d065fb1e7  guix-build-a24ef27cfe6e/output/riscv64-linux-gnu/SHA256SUMS.part
f037d4cc378da0c00b10f2960cb853377b2c0710aba4ed17c8f10ff9e8ab2cd8  guix-build-a24ef27cfe6e/output/riscv64-linux-gnu/bitcoin-a24ef27cfe6e-riscv64-linux-gnu-debug.tar.gz
793d0f8710e300adb6aff2e289370fb71c60d5d844970161898c359f3dea0f2f  guix-build-a24ef27cfe6e/output/riscv64-linux-gnu/bitcoin-a24ef27cfe6e-riscv64-linux-gnu.tar.gz
caeed20b8b301abdd32cb82c290ef77b283eb7b9f341254cdcf846f6d85365a9  guix-build-a24ef27cfe6e/output/x86_64-apple-darwin/SHA256SUMS.part
5f02de0e7149b70b6fe612cc382e27c3e8691a2c8d05730604ea92410843d62c  guix-build-a24ef27cfe6e/output/x86_64-apple-darwin/bitcoin-a24ef27cfe6e-x86_64-apple-darwin-unsigned.dmg
0f5a5041a12061e4e9f5fcfc803cd2a98636d964845a76713d83575caef5a4f6  guix-build-a24ef27cfe6e/output/x86_64-apple-darwin/bitcoin-a24ef27cfe6e-x86_64-apple-darwin-unsigned.tar.gz
adc74b896eba9eb6b6ef5da0d909f2d3c529a23b55fff4c8af4465b277b46217  guix-build-a24ef27cfe6e/output/x86_64-apple-darwin/bitcoin-a24ef27cfe6e-x86_64-apple-darwin.tar.gz
57a56059e4054e5becdbc36a6765f3b209535a78d4fa3f8a932d65e6b12d9c05  guix-build-a24ef27cfe6e/output/x86_64-linux-gnu/SHA256SUMS.part
5c163f6c9936307b139fcc91e9a129e144c7961b44e11b245b0c9402b6b919a8  guix-build-a24ef27cfe6e/output/x86_64-linux-gnu/bitcoin-a24ef27cfe6e-x86_64-linux-gnu-debug.tar.gz
7c1f9370fe8e26e3363198f3cd3f1d817662bc944f873138bcff997cda2d3c25  guix-build-a24ef27cfe6e/output/x86_64-linux-gnu/bitcoin-a24ef27cfe6e-x86_64-linux-gnu.tar.gz
0708072d2c81ed5e7d54bfec350d8737931e13bd9291bdb72925c096be33f186  guix-build-a24ef27cfe6e/output/x86_64-w64-mingw32/SHA256SUMS.part
0b9b931fb4902a2779a193ca96fee01d1790644a004cd8898e24f008799d1a24  guix-build-a24ef27cfe6e/output/x86_64-w64-mingw32/bitcoin-a24ef27cfe6e-win64-debug.zip
d19c8cbb2b3359ebf0b66dada69f3a88eb41c0945c0e2b847bc2684f1c3d1985  guix-build-a24ef27cfe6e/output/x86_64-w64-mingw32/bitcoin-a24ef27cfe6e-win64-setup-unsigned.exe
5797bb51c61b2857c0912e205bc886a29bc537076da00bba4c3e27c479d23c9c  guix-build-a24ef27cfe6e/output/x86_64-w64-mingw32/bitcoin-a24ef27cfe6e-win64-unsigned.tar.gz
131d9aff0d9c72f1d27cac728b7fdd42f6e961214eec330fc38e1d644869b6f8  guix-build-a24ef27cfe6e/output/x86_64-w64-mingw32/bitcoin-a24ef27cfe6e-win64.zip
```

**Review by hebasto — APPROVED** [review]
ACK a24ef27cfe6e28979cdc33382d69a979367febff

File: depends/packages/qt.mk [hunk 1]
```diff
@@ -13,7 +13,7 @@ $(package)_patches += dont_use_avx_android_x86_64.patch dont_hardcode_x86_64.pat
 $(package)_patches += fix_android_jni_static.patch dont_hardcode_pwd.patch
 $(package)_patches += qtbase-moc-ignore-gcc-macro.patch fix_limits_header.patch
 $(package)_patches += fix_bigsur_style.patch use_android_ndk23.patch
-$(package)_patches += rcc_hardcode_timestamp.patch
+$(package)_patches += rcc_hardcode_timestamp.patch duplicate_lcqpafonts.patch
```
Comment (prusnak) [inline depends/packages/qt.mk:16]: Can we use one patch per line next time, please?

**Pull request merged**